samples/cellular/modem_shell/src/sock/sock_shell.c

```diff
@@ -27,36 +27,21 @@
 /* Maximum length of the address */
 #define SOCK_MAX_ADDR_LEN 100
 
-enum sock_command {
+enum sock_shell_command {
 	SOCK_CMD_CONNECT = 0,
+	SOCK_CMD_CLOSE,
 	SOCK_CMD_SEND,
 	SOCK_CMD_RECV,
-	SOCK_CMD_CLOSE,
-	SOCK_CMD_RAI,
-	SOCK_CMD_LIST
+	SOCK_CMD_RAI
 };
 
-static const char sock_usage_str[] =
-	"Usage: sock <subcommand> [options]\n"
-	"\n"
-	"Subcommands:\n"
-	"  connect:  Open socket and connect to given host. No mandatory options.\n"
-	"  close:    Close socket connection. Mandatory options: -i\n"
-	"  send:     Send data. Mandatory options: -i\n"
-	"  recv:     Initialize and query receive throughput metrics. Without -r option,\n"
-	"            returns current metrics so that can be used both as status request\n"
-	"            and final summary for receiving.\n"
-	"            Mandatory options: -i\n"
-	"  rai:      Set Release Assistance Indication (RAI) parameters.\n"
-	"            In order to use RAI options for a socket, RAI feature must be\n"
-	"            globally enabled with 'link rai' command.\n"
-	"  list:     List open sockets. No options available.\n"
-	"\n"
-	"General options:\n"
+static const char sock_connect_usage_str[] =
+	"Usage: sock connect -i <socket id> -a <address> -p <port>\n"
+	"       [-f <family>] [-t <type>] [-b <port>] [-I <cid>]\n"
+	"       [-S] [-T <sec_tag>] [-c] [-V <level>] [-H <hostname>]\n"
+	"Options:\n"
 	"  -i, --id, [int]           Socket id. Use 'sock list' command to see open\n"
```

Comment:
I think `-i` is not used with `sock connect`.

Reply:
True. Removed here, usage and getopt loop.